packages/plugin-ngrok/project.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "plugin-ngrok",
```

Comment:
The graph looks accurate:
<img width="204" alt="image" src="https://user-images.githubusercontent.com/663605/188117140-e2a4f1bf-c339-41c0-af48-6f2850268e60.png">